internal/fixtures/model.go

```diff
@@ -97,6 +97,17 @@ education:
     location: Lens, France
     dates: 2019 - 2014
     link: https://ig2i.centralelille.fr
+
+references:
+  - name: Jane Smith
+    position: CTO
+    company: Tech Innovations
+    date: Janvier 2024
+    linkedin: janesmith
```

Comment:
After introducing `socialNetworks`, you can remove the `linkedin` attribute.

Do this in files:

- `cv.yml` at line 500
- `internal/fixtures/model.go` at line 247
- `internal/model/model.go` ar line 22